Api/Controllers/CharactersController.cs

```diff
@@ -40,23 +36,11 @@ public CharactersController(IDbContext context, IMediator mediator)
         // Returns a list of all your Characters
         //
         [HttpGet]
-        public async Task<IActionResult> GetCharacters([FromQuery] string filter)
+        public async Task<ActionResult<IEnumerable<GetCharactersResponse>>> GetCharacters([FromQuery] string filter)
         {
             var response = await _mediator.Send(new GetCharactersRequest { Filter = filter });
 
-            switch (response.ResponseStatus)
-            {
-                case ResponseStatus.Ok:
-                    return Ok(response.Data);
-                case ResponseStatus.NotFound:
-                    return NotFound(new { errors = new List<string>() { response.ResponseMessage } });
-                case ResponseStatus.BadRequest:
-                    return BadRequest(new { errors = new List<string>() { response.ResponseMessage } });
-                case ResponseStatus.NotAuthorized:
-                    return Forbid();
-                default:
-                    return StatusCode(500, new { errors = new List<string>() { response.ResponseMessage } });
-            }
+            return Ok(response);
```

Comment:
Quick shorthand for all of these changed controllers btw:
`public async Task<>...... ([FromQuery] string filter)
    => Ok(await _mediator.send(.....)`